pkgs/by-name/ka/kanha/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  rustPlatform,
+  fetchCrate,
+  stdenv,
+  darwin,
+  pkgs,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kanha";
+  version = "0.1.2";
+
+  src = fetchCrate {
+    inherit version;
+    pname = "kanha";
+    hash = "sha256-ftTmYCkra3x/oDgGJ2WSf6yLeKXkwLJXhjuBdv7fVLY=";
+  };
+
+  cargoHash = "sha256-kjHLc+qWo5dB4qbdlIWzk/pjpghRaDcX/7kkjEM219c=";
+
+  buildInputs =
+    with pkgs;
```

Comment:
```suggestion
```

Reply:
I've made the required changes and force pushed with the same commit, Could you now please review one?